cli/src/lib.rs

```diff
@@ -246,13 +248,25 @@ impl Iroha {
             std::path::Path::new(&config.kura.block_store_path),
             config.kura.debug_output_new_blocks,
         )?;
+        let live_query_store_handle =
+            LiveQueryStore::from_configuration(config.live_query_store).start();
```

Comment:
did you think about making query store shared between client and smart contracts? I just wonder if you have any thoughts around that